### PR TITLE
[documentation] Clarify privateKeyToAccount private string is hex

### PR DESCRIPTION
**Pull request by tcrowe**
## What does it do?

#1267 brought up a good point that `web3.eth.accounts.privateKeyToAccount` `privateKey` does not specify that it's a hex string. This PR fixes that in the documentation.

## Does it work?

screen shot 2018-12-15 at 4 50 09 pm

### CODE REVIEW

**Review by joshstevens19 — APPROVED** [review]

**Comment by joshstevens19** [comment]
Thanks @tcrowe, looks fine to me, we need 2 approvals before we can merge so @nivida can you merge when you get a second. 

Just a note @tcrowe  if your doing doc updates can you please do it against ethereumProvider branch as that will be the new 1.0.x and default branch once its deployed.

**Comment by nivida** [comment]
@joshstevens19 That's wrong they don't have to be against the ethereumProvider because I didn't change the documentation there and I will merge the ethereumProvider branch into the 1.0 branch.

@tcrowe Thanks for submitting this PR! :)

**Comment by joshstevens19** [comment]
Surely you want to keep your `ethereumProvider` inline with the latest code so would of assumed you would want it in the `ethereumProvider`, sorry if that was incorrect 👍

**Pull request merged**
